filenames.gypi

```diff
@@ -184,6 +187,10 @@
       'atom/browser/atom_browser_context.h',
       'atom/browser/atom_download_manager_delegate.cc',
       'atom/browser/atom_download_manager_delegate.h',
+      'atom/browser/in_app_purchase.h',
+      'atom/browser/in_app_purchase_mac.mm',
+      'atom/browser/in_app_purchase_observer.h',
+      'atom/browser/in_app_purchase_observer_mac.mm',
       'atom/browser/atom_browser_main_parts.cc',
       'atom/browser/atom_browser_main_parts.h',
       'atom/browser/atom_browser_main_parts_mac.mm',
```

Comment:
This feature will never be used on any platform other than macOS,
can you please put all of its files under a OS check here? Right next to the `['OS=="win"', {` section.